apps/docs/src/app/core/component-docs/info-label/info-label-docs.component.html

```diff
@@ -30,7 +30,7 @@
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'filled'" [componentName]="'infolabel'">
+<fd-docs-section-title [id]="'infoLabelNumeric'" [componentName]="'infolabel'">
 	Info label with numeric and only Icon
 </fd-docs-section-title>
 <description> <code>fd-info-label--numeric fd-info-label--only-icon</code> modifier can be used to render a numeric and icon only version of the badge. </description>
```

Comment:
The `fd-info-label--numeric` and `fd-info-label--only-icon` modifiers can be used to render numeric-only and icon-only versions of the badge.